src/fastoad/cmd/api.py

```diff
@@ -180,21 +181,37 @@ def list_variables(
         # For a terminal output, we limit width of NAME column
         variables_df["NAME"] = variables_df["NAME"].apply(lambda s: "\n".join(tw.wrap(s, 50)))
```

Comment:
I have the same behaviour too. I do not know why it happens with the var_desc format.
I am not against removing this line. Did you mature this idea?

Reply:
It simply happens because this code line separates the variable name on two lines, and then `.to_csv()` has to handle that.

We could do this wrapping if format is not var_desc, but yes, I think we should remove it.
Anyway, with large variable names, file output is already preferable to terminal output.

Reply:
I agree. Corrected in be994a1.